### DISCUSSION TITLE
Why do I not see a command line prompt?

### DISCUSSION THREAD

**Discussion by eimrine** [janet-lang/janet#1080] [Q&A] [answered]
I am interested in the language. I have installed it in Windows 7 x64 from installer and opened the binary.
I see "Janet 1.26.0-local windows/x64 - '(doc)' for help" line and nothing else, but I expect to see a prompt.
Also I can not type any symbol when I am running Janet binary.
I tried double-click the binary or run it from cmd and the outcome is same.
I can run some Janet code via janet -e, also I have a different OS but if it is possible I want Janet REPL working on this machine.
Any help?

**Comment by bakpakin** [comment, marked as answer]
The short answer is that the janet uses win32 apis to setup the repl that apparently don't work well in windows 7. The quick fix solution is to use windows 10 (utf-8 support in the cmd.exe terminal window), but this may be a bug. Windows 7 is now EOL and I do not test on windows 7.
